graphdatascience/session/dedicated_sessions.py

```diff
@@ -44,19 +45,28 @@ def get_or_create(
         dbid = AuraApi.extract_id(db_connection.uri)
         existing_session = self._find_existing_session(session_name, dbid)
 
+        # hashing the password to avoid storing the actual db password in Aura
+        password = hashlib.md5(db_connection.password.encode()).hexdigest()
```

Comment:
nit: md5 is generally not recommended as hasing function. I think the reasoning does not apply here (missing salt while hashing for example), but still: Do we have the option of using another algorithm here?

Reply:
we could totally use another one. need to make sure it also exists in other languages for later APIs.
But we cannot use salting here as we want the determinism. (the user would need to provide the salt, which is no different to a password at this point).

sha256 looks like an option